Framework/Source/Graphics/Model/Loaders/AssimpModelImporter.h

```diff
@@ -87,7 +88,13 @@ namespace Falcor
         void loadTextures(const aiMaterial* pAiMaterial, const std::string& folder, BasicMaterial* pMaterial, bool isObjFile, bool useSrgb);
         Material::SharedPtr createMaterial(const aiMaterial* pAiMaterial, const std::string& folder, bool isObjFile, bool useSrgb);
 
+        // Checks whether a node or its name corresponds to a used bone or node in the skeleton hierarchy
+        bool isUsedNode(const aiNode* pNode) const;
+
         std::map<std::string, uint32_t> mBoneNameToIdMap;
+        // Non-bone nodes need to be counted by pointer because they can have duplicate names after assimp processing (e.g. "RootNode")
+        std::unordered_set<const aiNode*> mAdditionalUsedNodes;
```

Comment:
Do we need this after load time? If not, just pass it around when loading data.

Reply:
We don't need anything in the importer after load time. Importer instances only exist when static `import()` is called.